pkg/sqs/SqsConnectionFactory.php

```diff
@@ -120,10 +125,7 @@ private function parseDsn(string $dsn): array
         $dsn = Dsn::parseFirst($dsn);
 
         if ('sqs' !== $dsn->getSchemeProtocol()) {
-            throw new \LogicException(sprintf(
-                'The given scheme protocol "%s" is not supported. It must be "sqs"',
-                $dsn->getSchemeProtocol()
-            ));
+            throw new \LogicException(sprintf('The given scheme protocol "%s" is not supported. It must be "sqs"', $dsn->getSchemeProtocol()));
```

Comment:
This is only a code style change, and it violates the soft limit for characters in a single row (80-120, usually, see PSR https://www.php-fig.org/psr/psr-2/#23-lines).

Reply:
Hi and thanks for your fast feedback :+1: . No problem i'll revert this in few minutes. 

Perhaps their is a missconfiguration with what's described in the https://github.com/php-enqueue/enqueue-dev/blob/master/docs/contribution.md guide because those lines were automatically updated by php-cs fixer.

Reply:
If it's true then keep it, otherwise CI will complain when testing. We will need to update php-cs ruleset then.

Reply:
Ok, you're right I tested it with the change on one line and it make the CI fail. So I revert it to 3 lines.